wandb/sdk/wandb_run.py

```diff
@@ -3180,6 +3186,27 @@ def _footer_local_warn(
                     status="warn",
                 )
 
+    @staticmethod
+    def _footer_server_messages(
+        poll_exit_response: Optional[PollExitResponse] = None,
+        quiet: Optional[bool] = None,
+        *,
+        settings: "Settings",
+        printer: Union["PrinterTerm", "PrinterJupyter"],
+    ) -> None:
+
+        if (quiet or settings.quiet) or settings.silent:
+            return
+
+        if settings.disable_hints:
+            return
+
+        if poll_exit_response and poll_exit_response.server_messages:
+            for message in poll_exit_response.server_messages.item:
+                text = message.html_text if printer._html else message.utf_text
```

Comment:
I think you need to check message type here, and probably level if we ever want to add other hints.

Maybe messagetype of "footer" and level of?

Reply:
Yeah i didn't add this logic since I didn't know how we are going to use it... (i assumed we will add it once we actually have the categories we want to use)